### DISCUSSION TITLE
RCS icon

### DISCUSSION THREAD

**Discussion by samarthpuri** [twilio-labs/paste#3974] [Request: Icon] [answered]
We shipped out RCS in Messaging Service using [this icon from Paste](https://www.figma.com/file/l2qtpQVMmXzQPVB3rEAA8i/Paste-Components-%26-Themes?type=design&node-id=15809%3A96702&mode=design&t=3N1fTrQymTvHetZz-1) since it's device specific a few quarters ago. Do we need to update this and request a RCS specific icon, or can we keep it as is?

**Comment by serifluous** [comment]
Hey @samarthpuri, can you share the Figma file or screenshots where you're using the icon so we can understand the context?

**Comment by samarthpuri** [comment]
[Figma](https://www.figma.com/design/4LoSo1G7dBh9ebjyvH5Git/2023-Q4-RCS?node-id=337-34834&t=K7zfxyDqVMVRvZ5H-1)
![RBM senders search](https://github.com/twilio-labs/paste/assets/23617133/6ac3c981-8be4-4406-afc3-90883f21c0b8)

**Reply by serifluous** [reply]
To help point you to the right icon, in this context, does the icon mean SMS-capable, RCS-capable, message-template-capable, or something else? And is this icon the only one that would ever show in this column?

Regardless of the icon, though, I'd recommend pairing text with it, or updating the column header text, so it's clearer upfront what it means.

**Comment by samarthpuri** [comment]
The icon means RCS capable. In the context of RCS, this would be the only icon shown in the capabilities column

Icons shown in Messaging Services are based on the sender type. I've attached an example for phone numbers which show SMS and MMS icon capabilities. Other sender types include Facebook Messenger for Business and WhatsApp which have their respective corporate icons 

For context ,RCS capabilities depends on the mobile phone you're using (e.g. you have an Android phone which allows you to send/receive RCS messages)

<img width="1500" alt="Screenshot 2024-07-05 at 1 45 03 PM" src="https://github.com/twilio-labs/paste/assets/23617133/3bd79863-4bfa-4246-a795-543a51126f0c">

**Reply by serifluous** [reply]
ok cool, is an RCS icon supposed to show up in the 2nd screenshot you sent too, if the sender is RCS-capable? If so, I'd suggest requesting an icon specifically for RCS capabilities then. Dependency request form here: https://airtable.com/appwvsQKyJHfnG5Sa/shrvMvL3BG6Zm0f9l And let us know what you think would work well for the glyph.

For the first screenshot, do you need the "Capabilities" column at all? Do only RCS-capable senders show up in that list?

**Reply by samarthpuri** [reply]
The RCS icon wouldn't appear in the second screenshot because it focuses on selecting phone numbers for a sender pool. However, if they selected add an RCS sender, they'll see the RCS icon as shown in screenshot 1.

The 'Capabilities' column isn't needed when adding RCS. However, once you've confirmed your sender pool with different sender types, the icons in the 'Capabilities' column will be used to distinguish between them

**Reply by serifluous** [reply, marked as answer]
Got it, I'd definitely recommend requesting an RCS-specific icon then ^, and still pairing the icon with text.

**Comment by serifluous** [comment]
Hey @samarthpuri  ! The icon (RCSCapable) is now available in Paste.
